src/lib.rs

```diff
@@ -80,6 +82,8 @@ mod codegen {
     include!(concat!(env!("OUT_DIR"), "/codegen.rs"));
 }
 
+pub use features::{RustTarget, LATEST_STABLE_RUST, RUST_TARGET_STRINGS};
```

Comment:
I don't see `RUST_TARGET_STRINGS` used in this module, is it a leftover from some earlier iteration?